.coveragerc

```diff
@@ -700,11 +700,7 @@ omit =
     homeassistant/components/omnilogic/__init__.py
     homeassistant/components/omnilogic/common.py
     homeassistant/components/omnilogic/sensor.py
-    homeassistant/components/ondilo_ico/__init__.py
-    homeassistant/components/ondilo_ico/api.py
-    homeassistant/components/ondilo_ico/const.py
-    homeassistant/components/ondilo_ico/oauth_impl.py
-    homeassistant/components/ondilo_ico/sensor.py
```

Comment:
Why are the ondilo files removed from coverage?

Reply:
Not sure. I just rebased you the latest version merged so this was current.

Reply:
OK, something must have gone wrong in the merge/rebase then. Since the commits will be squashed, you can just add another commit re-adding those lines.

Reply:
OK will re-add